_data/navigation.yml

```diff
@@ -14,10 +14,10 @@
   url: /guides/
   icon: '📖'
   children:
-    - title: Using a remote index
+    - title: Using remote index
```

Comment:
i feel like `a` in there is actually okay, as it conveys the fact that there can be multiple instances and the guide talks about how to connect to one in general.